<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>search_widget</class>
 <widget class="QWidget" name="search_widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>427</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tab_search">
     <property name="focusPolicy">
      <enum>Qt::TabFocus</enum>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::NoContextMenu</enum>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="tabShape">
      <enum>QTabWidget::Triangular</enum>
     </property>
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="pioneers">
      <attribute name="title">
       <string>Pioneers</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLineEdit" name="lineEdit_pioneers">
         <property name="text">
          <string/>
         </property>
         <property name="frame">
          <bool>false</bool>
         </property>
         <property name="placeholderText">
          <string>Search pioneers...</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QTableView" name="tableView_pioneers">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="computers">
      <attribute name="title">
       <string>Computers</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLineEdit" name="lineEdit_computers">
         <property name="frame">
          <bool>false</bool>
         </property>
         <property name="placeholderText">
          <string>search computers...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableView" name="tableView_computers">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="connections">
      <attribute name="title">
       <string>Connections</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLineEdit" name="lineEdit_connections">
         <property name="frame">
          <bool>false</bool>
         </property>
         <property name="placeholderText">
          <string>Search connections...</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QTableView" name="tableView_connections">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
